numerapi/base_api.py

```diff
@@ -444,3 +445,47 @@ def get_transactions(self, model_id: str = None) -> Dict:
             utils.replace(t, "value", utils.parse_float_string)
             utils.replace(t, "insertedAt", utils.parse_datetime_string)
         return txs
+
+    def get_current_round(self, tournament=8):
```

Comment:
Good idea moving `get_current_round` to the base class. However, I don't want to bother users with remembering tournament numbers. To avoid that, please follow how the `get_models` method works, and return the current round number depending on the used API (numerapi vs signalsapi)